src/sentry/tasks/backfill_seer_grouping_records.py

```diff
@@ -186,14 +198,18 @@ def backfill_seer_grouping_records(
                         }
                     }
             if not dry_run:
-                Group.objects.bulk_update(groups, ["data"])
+                num_updated = Group.objects.bulk_update(groups, ["data"])
+                logger.info(
+                    "backfill_seer_grouping_records.bulk_update",
+                    extra={"project_id": project.id, "num_updated": num_updated},
+                )
 
         last_processed_id = group_id_message_data_batch[len(group_id_message_data_batch) - 1][0]
         redis_client.set(
             f"{make_backfill_redis_key(project_id)}",
             last_processed_id if last_processed_id is not None else 0,
             ex=60 * 60 * 24 * 7,
-        )  # needed for typing
+        )
```

Comment:
deleted this comment, not sure what it was referring to

Reply:
it was referring to this: `last_processed_id if last_processed_id is not None else 0`. cause we can pass in None, but we always will set it to 0 if it is None. but typing was getting mad. not a big deal to delete it though

Reply:
ah got it.